truss/tests/templates/control/control/test_server.py

```diff
@@ -17,6 +17,8 @@
     )
 )
 
+sys.path.append(str(Path(__file__).parent.parent.parent.parent.parent / "templates"))
```

Comment:
This isn't ideal. Maybe leave a comment explaining this line and what the expected directory structure is. When we refactor, this'll need to be updated as the directory structure will probably shift.